prov/shm/src/smr.h

```diff
@@ -320,6 +320,19 @@ static inline bool smr_vma_enabled(struct smr_ep *ep,
 			peer_smr->xpmem_cap_self == SMR_VMA_CAP_ON);
 }
 
+static inline void smr_set_ipc_invalid(struct smr_region *region, uint64_t id)
+{
+	if (region->map->peers[id].pid_fd == -1)
```

Comment:
Since the pid_fd gets initialized to -1 for non ZE cases, do we accidentally disable the other iface IPCs?

Do we need to add an if ZE check to conditionally set the validity?
```
static inline void smr_set_ipc_invalid(struct smr_region *region, uint64_t id)
{
	if (ofi_hmem_initialized(FI_HMEM_ZE) && region->map->peers[id].pid_fd == -1)
		smr_peer_data(region)[id].ipc_invalid = 1;
}
```